app/forms/pages/type_of_answer_form.rb

```diff
@@ -4,15 +4,32 @@ class Pages::TypeOfAnswerForm < BaseForm
   validates :draft_question, presence: true
   validates :answer_type, presence: true, inclusion: { in: Page::ANSWER_TYPES }
 
-  def submit(session)
+  def submit
     return false if invalid?
+    return true unless answer_type_changed?
 
     draft_question
-      .assign_attributes({ answer_type:, answer_settings: nil })
+      .assign_attributes({ answer_type:, answer_settings: default_answer_settings_for_answer_type })
```

Comment:
This is a huge issue we currently have. if a user changes their answer_type or in some cases only clicked "continue" we would completely remove all existing answer settings.

This feels like a better approach because we know what default settings each answer type should have. This isn't perfect but I'll explain why during Thursday's mod session.